samples/README.md

```diff
@@ -67,18 +74,30 @@ To run the word count sample in Dataproc cluster, follow the steps:
    gcloud config set dataproc/region $REGION
    ```
 <!-- TODO: set up bots to update jar version -->
-9. Run the sample in Dataproc. You would see the word count result show up in the console output.
+9. Run the sample in Dataproc. This will perform word count aggregation and publish word count results to Pub/Sub Lite.
    ```sh
    gcloud dataproc jobs submit spark --cluster=$CLUSTER_NAME \
       --jars=$BUCKET/pubsublite-spark-snippets-$SAMPLE_VERSION.jar,$BUCKET/pubsublite-spark-sql-streaming-$CONNECTOR_VERSION-with-dependencies.jar \
```

Comment:
If you create a Uber or Fat Jar, you can just pass the sample jar - it's a bit easier.

Reply:
https://stackoverflow.com/questions/11947037/what-is-an-uber-jar

Reply:
The samples and connector code inherited from two different pom parents: 
samples <- com.google.cloud.samples.shared-configuration
connector <- com.google.cloud.google-cloud-pubsublite-parent
The connector-with-dependencies is itself a fat jar, and I don't think it's a good idea to link the connector code into samples.
Also, I would rather they use the sample with a released connector instead of building from the repo directly.